client/landing/gutenboarding/onboarding-block/create-site/style.scss

```diff
@@ -35,38 +35,27 @@ $progress-widget-height: 300px;
 		padding: 1em;
 
 		.create-site__progress {
-			height: $progress-step-title-height;
-			overflow: hidden;
 			margin-bottom: 20px;
 		}
 
-		.create-site__progress-steps {
-			animation: create-site__progress-steps $progress-duration ease-out forwards;
-
-			@keyframes create-site__progress-steps {
-				@for $i from 0 through 3 {
-					#{$i / 3 * 100%} {
-						transform: translateY( -$progress-step-title-height * ( $i - 1 ) );
-					}
-				}
-			}
-		}
-
 		.create-site__progress-bar {
-			height: 7px;
-			background: var( --mainColor );
-			transform: scaleX( 0 );
-			transform-origin: 0% 0%;
+			position: relative;
+			overflow: hidden;
+			height: 6px;
 			margin-top: 1em;
+			background: var( --studio-gray-10 );
+			--progress: 0;
 
-			animation: create-site__progress-bar-progress-move #{$progress-duration + 1} ease-out forwards;
-
-			@keyframes create-site__progress-bar-progress-move {
-				@for $i from 1 through 3 {
-					#{$i / 3 * 100%} {
-						transform: scaleX( #{$i / 3} );
-					}
-				}
+			&::before {
+				background: var( --mainColor );
+				transform: translateX( calc( -100% * min( 1 - var( --progress, 0 ), 1 ) ) );
```

Comment:
Should we have some fallback for IE11?

Reply:
🤔 We can just set the transform style in React I suppose.

Reply:
It's a `before` element, so I can't actually 😛 

I'm glad there was a reason I decided to juggle these CSS custom properties initially. I think no moving progress bar is an acceptable graceful degradation for IE11.